sims/verilator/Makefile

```diff
@@ -86,7 +86,9 @@ ARIANE_VERILATOR_FLAGS = \
 CHIPYARD_VERILATOR_FLAGS = \
 	--assert
 
+# Use --timescale to approximate timescale behavior of pre-4.034
 VERILATOR_NONCC_OPTS = \
+	--timescale 1ns/1ps \
```

Comment:
We should consider having the argument part of this pulled out since we give the same value to both verilator and vcs.

Reply:
I could just put it in `SIM_FLAGS`?

Reply:
Unfortunately one of them wants 2 dashes and the other wants 1 so I was only suggestions lifting the standard verilog `1ns/1ps`.
Its too much work for now but it would be nice to keep things DRY.

Reply:
`$(SIM_FLAG_DASHES)timescale 1ns/1ps` :)

Reply:
wonderful :)

Reply:
Where's that chef-albert garnishing gif when you need it.